### PR TITLE
Add support for Fedora 35

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Also, set fixed version for F34.

~~DO NOT MERGE YET, DOES NOT WORK YET. The site-packages path is wrong on the f35-devel & -release branches of anaconda repo.~~

### CODE REVIEW

File: anaconda_updates/scripts/show_version.sh [hunk 1]
```diff
@@ -74,6 +74,10 @@ do
          PACKAGES_URL="fedora34/Packages/a"
          shift
          ;;
+        -f35)
+         PACKAGES_URL="fedora_devel/Packages/a"
```
Comment (VladimirSlavik) [inline anaconda_updates/scripts/show_version.sh]: Of note: Path to packages is different on new cobra02.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

File: anaconda_updates/scripts/show_version.sh (hunk 1 above)
Comment (jkonecny12) [reply anaconda_updates/scripts/show_version.sh]: This script shouldn't really reflect cobra02 because it's not publicly accessible.

**Review by poncovka — APPROVED** [review]

File: anaconda_updates/scripts/show_version.sh (hunk 1 above)
Comment (VladimirSlavik) [reply anaconda_updates/scripts/show_version.sh]: OK, but then what should it reflect? The server at the top is nonsensical, so the paths make zero sense anyway.

File: anaconda_updates/anaconda_updates/releases/f35.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+class Fedora35Branch(GeneralBranch):
+
+    def __init__(self):
+        super().__init__(branch_type=Branch.fedora35,
+                         cmd_args=["-f35", "--fedora35"],
+                         help="working on Fedora 35",
+                         version_script_params=["-f35", "-p"],
+                         site_packages="./usr/lib/python3.9/site-packages/")
```
Comment (poncovka) [inline anaconda_updates/anaconda_updates/releases/f35.py]: I have checked that and it is 3.10. The support for master needs to be updated as well.

Comment (VladimirSlavik) [reply anaconda_updates/anaconda_updates/releases/f35.py]: Thank you, I missed that. For the record, the values here are actually not used for anaconda, so things worked just fine...

**Review by poncovka — APPROVED** [review]
Thank you!

**Pull request merged**